src/PKSim.Core/Snapshots/Mappers/SnapshotMapper.cs

```diff
@@ -7,6 +7,37 @@
 
 namespace PKSim.Core.Snapshots.Mappers
 {
+   public class SnapshotContext
```

Comment:
@Yuri05 This is the big change. A context pass to every mapper when going from snapshot to model with the snapshot version and the project being currently updated